management/src/test/java/org/kaazing/gateway/management/jmx/JmxRoundTripLatencyIT.java

```diff
@@ -119,22 +124,23 @@ public void getRoundTripLatencyAttributesFromJmx() throws Exception {
 
         MBeanServerConnection mbeanServerConn = jmxConnection.getConnection();
         Set<ObjectName> mbeanNames = mbeanServerConn.queryNames(null, null);
-        String MBeanPrefix = "subtype=services,serviceType=echo,serviceId=\"" + WS_URI + "\",name=sessions";
+        String MBeanPrefix = "subtype=services,serviceType=echo,serviceId=\"" + ECHO_WSN_SERVICE + "\",name=sessions";
         for (ObjectName name : mbeanNames) {
             if (name.toString().indexOf(MBeanPrefix) > 0) {
+                mbeanServerConn.getObjectInstance(name);
```

Comment:
I think so, though I'm no JMX expert.  In the debugger this invoked the RMI to get the instance locally, the following commands queried it